tests/fetchStats.test.js

```diff
@@ -88,12 +91,28 @@ const error = {
 
 const mock = new MockAdapter(axios);
 
+/**
+ * Mocks the GraphQL API based on certain conditions.
+ *
+ * @param {*} config Axios config object.
+ * @returns Axios response object.
+ */
```

Comment:
Looks like this js-doc should not be there

Reply:
correct, removed.